tools/external/tools.go

```diff
@@ -3,8 +3,5 @@
 package tools
 
 import (
-	_ "github.com/AlekSi/gocoverutil"
 	_ "github.com/golang/protobuf/protoc-gen-go"
```

Comment:
whoops. this should be protoc-gen-doc, since that is the only bit left that is built out-of-tree